modules/sizing-requirements-for-gitops.adoc

```diff
@@ -0,0 +1,29 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/about-sizing-requirements-gitops.adoc
+
+:_content-type: CONCEPT
+[id="sizing-requirements-for-gitops_{context}"]
+= Sizing requirements for GitOps
+
+[role="_abstract"]
+{gitops-title} is a declarative way to implement continuous deployment for cloud-native applications. Through GitOps, you can define and configure the CPU and memory requirements of your application. Every time you install the {gitops-title} Operator, the resources on the namespace are installed within the defined limits. If the default install does not set any limits or requests, the Operator fails within the namespace which has quotas. Without enough resources, the cluster is unable to schedule ArgoCD related pods. The following table details the resource requests and limits for the default workloads:
```

Comment:
Would it help to separate the abstract from the longer explanation?

Reply:
```suggestion
{gitops-title} is a declarative way to implement continuous deployment for cloud-native applications. Through GitOps, you can define and configure the CPU and memory requirements of your application.

 Every time you install the {gitops-title} Operator, the resources on the namespace are installed within the defined limits. If the default install does not set any limits or requests, the Operator fails within the namespace which has quotas. Without enough resources, the cluster is unable to schedule ArgoCD related pods. The following table details the resource requests and limits for the default workloads:
```

Reply:
In light of the IBM SG guidance in the "Relative Pronouns" section, would it make sense to review your usage of "which" and "that" throughout this PR?

Reply:
`is unable to` > `cannot`

Reply:
> In light of the IBM SG guidance in the "Relative Pronouns" section, would it make sense to review your usage of "which" and "that" throughout this PR?

This issue has not been resolved.